src/nft/components/collection/Filters.css.ts

```diff
@@ -8,7 +8,8 @@ export const container = style([
     paddingTop: '4',
   }),
   {
-    width: '300px',
+    width: '308px',
+    paddingRight: '8px',
```

Comment:
the shadow on the right slider overlaps the adjacent div, which creates a line to the right that looks strange(light mode only).  This maintains the original dimensions while solving that issue